terraform/lamp-stack-ibmcloud/scripts/setup_env.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash -xe 
+VAR_DB_PASSWORD="${VAR_DB_PASSWORD:-password}"
+# wait for setup
+while ! grep "Cloud-init .* finished" /var/log/cloud-init.log; do
+    echo "$(date -Ins) Waiting for cloud-init to finish"
+    sleep 2
+done
+until sudo apt-get update; do sleep 5; done # some hacky way to get apt-get to succeed at boot 
+# install apache mariadb php 
+sudo apt-get install software-properties-common -y
+sudo apt-key adv --recv-keys --keyserver hkp://keyserver.ubuntu.com:80 0xF1656F24C74CD1D8
+sudo add-apt-repository 'deb [arch=amd64,arm64,ppc64el] http://mirror.i3d.net/pub/mariadb/repo/10.4/ubuntu bionic main'
```

Comment:
Is ppc64le not enabled by default in NG Cloud?

Reply:
Correct. I don't need to add repo.